scipy/special/_basic.py

```diff
@@ -2421,12 +2421,14 @@ def factorial2(n, exact=False):
     array(105.00000000000001)
     >>> factorial2(7, exact=True)
     105
+    >>> factorial2(-10, exact=True)
```

Comment:
I almost wonder if this should be an error per my other comment on the double factorial of even negative integers, though I may have missed some of the literature.